### ISSUE TITLE
Segmentation fault on call

### ISSUE THREAD

**Issue by nbonacchi** [spyder-ide/spyder#5847]
At first I though it was some weird reflection call for the autocomplete/documentation feature when it scans the modules for the __doc__ bu the error seems to do with the GPU? 

`[27956:28000:1128/124044.495232:ERROR:browser_gpu_channel_host_factory.cc(108)] Failed to launch GPU process.
Created new window in existing browser session.
Segmentation fault (core dumped)`

**What steps will reproduce the problem?**

0. Create conda environment for python 3.5
1. open spyder
2. in the ipython console import matplotlib. pyplot as plt
3. type plt.plot() 
4. spyder disappears!

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.5.4
* Qt Versions: 5.6.2, PyQt5 5.6 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
psutil >=0.3     :  5.4.0 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by goanpeca** [comment]
@nbonachi, could you try settung a different default browser for your operating system?

**Comment by nbonacchi** [comment]
@goanpeca what do you mean other than chrome?

**Comment by goanpeca** [comment]
Well it depends. If something in spyder wanted to open a browser and the default is chrome or chromium this error might happen on some systems, that is why i suggested the change

**Comment by ccordoba12** [comment]
@nbonacchi, thanks for reporting. This is a duplicate of issue #5514 and it'll be fixed in our 3.2.5 version.
